app/controllers/concerns/solidus_wms/additional_orders_actions.rb

```diff
@@ -0,0 +1,33 @@
+require 'axlsx'
+
+module SolidusWms
+  module AdditionalOrdersActions
+    extend ActiveSupport::Concern
+
+    def export_xlsx
+      exporter = Rails.application.config.order_xls_export_class.new
```

Comment:
Please can we do this using an extension specific configuration, there is an example of how we've done this in the past in https://github.com/madetech/solidus_multi_domain_enhancements